### PR TITLE
Add log configuration items to the config file

### PR DESCRIPTION
**Pull request by SuperSupeng**
### What problem does this PR solve? 
Add log configuration items to the config file. #1384 

### What is changed and how it works?
Add logFileMaxSize, logFileMaxDays and logFileMaxBackups  items when load config file.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Release note
- no release note

### CODE REVIEW

File: cmd/server.go [hunk 1]
```diff
@@ -70,6 +70,9 @@ func initServerCmd(cmd *cobra.Command) {
 	cmd.Flags().Int64Var(&serverConfig.GcTTL, "gc-ttl", defaultServerConfig.GcTTL, "CDC GC safepoint TTL duration, specified in seconds")
 	cmd.Flags().StringVar(&serverConfig.LogFile, "log-file", defaultServerConfig.LogFile, "log file path")
 	cmd.Flags().StringVar(&serverConfig.LogLevel, "log-level", defaultServerConfig.LogLevel, "log level (etc: debug|info|warn|error)")
+	cmd.Flags().IntVar(&serverConfig.LogFileMaxSize, "log-file-max-size", defaultServerConfig.LogFileMaxSize, "max log file size in MB")
+	cmd.Flags().IntVar(&serverConfig.LogFileMaxDays, "log-file-max-days", defaultServerConfig.LogFileMaxDays, "max log file keep days")
+	cmd.Flags().IntVar(&serverConfig.LogFileMaxBackups, "log-file-max-backups", defaultServerConfig.LogFileMaxBackups, "maximum number of old log files to retain")
```
Comment (overvenus) [inline cmd/server.go:75]: I don't think we need to expose these settings in command flags.

It is enough to support these settings in TiCDC's config file, just like TiDB https://github.com/pingcap/tidb/blob/516584aaa47bec5a4dc131be95b413b48500f5fa/config/config.toml.example#L168-L179

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: cmd/server.go (hunk 1 above)
Comment (SuperSupeng) [reply cmd/server.go]: ok, I have modified it. PTAL

File: cmd/ticdc.toml [hunk 2]
```diff
@@ -15,6 +15,15 @@ log-file = "/tmp/ticdc/ticdc.log"
...
 log-level = "info"
 
+# Max log file size in MB (upper limit to 4096MB).
+log-file-max-size = 300
+
+# Max log file keep days. No clean up by default.
+log-file-max-days = 0
+
+# Maximum number of old log files to retain. No clean up by default.
+log-file-max-backups = 0
```
Comment (amyangfei) [inline cmd/ticdc.toml:25]: The log related configuration in TiDB/PD is different, should TiCDC keep the same config style?

/cc @leoppro 

```
[log]
# Log level: debug, info, warn, error, fatal.
level = "info"

[log.file]
# Log file name.
filename = ""

# Max log file size in MB (upper limit to 4096MB).
max-size = 300

# Max log file keep days. No clean up by default.
max-days = 0

# Maximum number of old log files to retain. No clean up by default.
max-backups = 0
```

Comment (ben1009) [reply cmd/ticdc.toml:25]: agree, rest lgtm

Comment (SuperSupeng) [reply cmd/ticdc.toml]: I have modified it. PTAL

Comment (overvenus) [reply cmd/ticdc.toml]: For the most cases, tiup sets log level and path via flags, so the compatibility issue is negligible.

**Review by overvenus — APPROVED** [review]
LGTM

File: cmd/server.go [hunk 3]
```diff
@@ -15,6 +15,7 @@ package cmd
 
 import (
 	"context"
+	"github.com/pingcap/ticdc/pkg/logutil"
```
Comment (overvenus) [inline cmd/server.go:18]: BTW, could you move the import to line 29?

Comment (SuperSupeng) [reply cmd/server.go]: get!

**Pull request closed without merging**